File: PRAK205-2510817210005-Daffa-Dziibaan-Khoiri.c
#include <stdio.h>
#include <math.h>

int main() {
    float alas, tinggi, keliling, luas, sisi_miring;

    scanf("%f %f", &tinggi, &sisi_miring);

    alas = sqrt(pow(sisi_miring, 2) - pow(tinggi, 2));
    keliling = tinggi + sisi_miring + alas;
    luas = (alas * tinggi) / 2;

    printf("Alas = %.0f cm\n", alas);
    printf("Tinggi = %.0f cm\n", tinggi);
    printf("Keliling = %.0f cm\n", keliling);
    printf("Luas = %.0f cm^2\n\n", luas);

    return 0;
}

File: PRAK205-2510817210005-Daffa-Dziibaan-Khoiri.py
import sys

input_data = sys.stdin.read().strip()
values = input_data.split()
tinggi, sisi_miring = map(float, values[:2])

alas = (sisi_miring**2 - tinggi**2) ** 0.5
keliling = tinggi + sisi_miring + alas
luas = (alas * tinggi) / 2
print(f"Alas    : {alas:.0f} cm")
print(f"Tinggi  : {tinggi:.0f} cm")
print(f"Keliling : {keliling:.0f} cm")
print(f"Luas    : {luas:.0f} cm^2")